File: app/src/main/java/com/likhit/variants/data/models/ExcludeListHelperModel.java
package com.likhit.variants.data.models;

public class ExcludeListHelperModel {

    private String fromGroupId;
    private String fromVariationId;
    private String toGroupId;
    private String toVariationId;

    public ExcludeListHelperModel() {
    }

    public ExcludeListHelperModel(ExcludeListHelperModel excludeListHelperModel) {
        this.fromGroupId = excludeListHelperModel.getToGroupId();
        this.fromVariationId = excludeListHelperModel.getToVariationId();
        this.toGroupId = excludeListHelperModel.getFromGroupId();
        this.toVariationId = excludeListHelperModel.getFromVariationId();
    }

    public String getFromGroupId() {
        return fromGroupId;
    }

    public void setFromGroupId(String fromGroupId) {
        this.fromGroupId = fromGroupId;
    }

    public String getFromVariationId() {
        return fromVariationId;
    }

    public void setFromVariationId(String fromVariationId) {
        this.fromVariationId = fromVariationId;
    }

    public String getToGroupId() {
        return toGroupId;
    }

    public void setToGroupId(String toGroupId) {
        this.toGroupId = toGroupId;
    }

    public String getToVariationId() {
        return toVariationId;
    }

    public void setToVariationId(String toVariationId) {
        this.toVariationId = toVariationId;
    }
}

File: app/src/main/java/com/likhit/variants/data/models/ExcludeListHelperModel.kt
package com.likhit.variants.data.models

class ExcludeListHelperModel {

    var fromGroupId: String? = null
    var fromVariationId: String? = null
    var toGroupId: String? = null
    var toVariationId: String? = null

    constructor() {}

    constructor(excludeListHelperModel: ExcludeListHelperModel) {
        this.fromGroupId = excludeListHelperModel.toGroupId
        this.fromVariationId = excludeListHelperModel.toVariationId
        this.toGroupId = excludeListHelperModel.fromGroupId
        this.toVariationId = excludeListHelperModel.fromVariationId
    }
}
